creating_citation_counts_tsv/requirements.txt

```diff
@@ -0,0 +1,7 @@
+findspark==1.4.2
+psutil==5.7.3
+py4j==0.10.9
+pyspark==3.0.1
+shortuuid==1.0.1
+tqdm==4.54.0
+wget==3.2
```

Comment:
`wget` is needed for the Bash scripts, but I haven't seen references to it in any Python code, so it seems unnecessary here:

```suggestion
```